spec/spec_helper_acceptance.rb

```diff
@@ -3,9 +3,9 @@
 configure_beaker do |host|
   case fact_on(host, 'operatingsystem')
   when 'CentOS'
-    host.install_package('epel-release')
+    install_module_from_forge_on(host, 'puppet/epel', '>= 3.0.0')
```

Comment:
previously, something complained about the missing upper limit. maybe that's gone now. updating this in the acceptancetests is always anoying

Reply:
The API it's using is `include epel`. It could probably work with `>= 0`.